src/browser/BrowserService.cpp

```diff
@@ -423,20 +423,23 @@ QJsonArray BrowserService::findMatchingEntries(const QString& id,
     return result;
 }
 
-void BrowserService::addEntry(const QString& id,
-                              const QString& login,
-                              const QString& password,
-                              const QString& url,
-                              const QString& submitUrl,
-                              const QString& realm,
-                              const QString& group,
-                              const QString& groupUuid,
-                              const QSharedPointer<Database>& selectedDb)
+BrowserService::ReturnValue
+BrowserService::addEntry(const QString& id,
+                         const QString& login,
+                         const QString& password,
+                         const QString& url,
+                         const QString& submitUrl,
+                         const QString& realm,
+                         const QString& group,
+                         const QString& groupUuid,
+                         const QSharedPointer<Database>& selectedDb)
 {
+    ReturnValue result = ReturnValue::Error;
```

Comment:
I don't understand why this variable is defined, it is not actually returned from the function.

Reply:
It's because of https://doc.qt.io/qt-5/qmetaobject.html#Q_RETURN_ARG.